test/UnitTest.cs

```diff
@@ -162,6 +179,39 @@ public void TestCSVReader()
             Assert.AreEqual("2,2,3,4", r[1]);
             Assert.AreEqual("5,6,7,8", r[2]);
         }
+
+        [TestMethod]
+        // it should exec after UBenchMarks
+        // the "row" in physicplan is stasticed based on the Hist or MCV, whitch is maintained in Catlog 
+        public void TestUnitTestConsistancy()
```

Comment:
I don't think we need a test just to verify consistency. Instead, you can insert the "clear stats" fix at the right place?